library/src/main/java/se/ingenuity/databinding/extension/ViewExt.kt

```diff
@@ -6,7 +6,7 @@ import androidx.annotation.IdRes
 fun View.findParentById(@IdRes id: Int): View? {
     var parent = this.parent
 
-    while ((parent as? View)?.id != id) {
+    while (parent != null && (parent as? View)?.id != id) {
```

Comment:
Note that `(parent as? View)?.id` effectively says `null != id`. Isn't that enough? Maybe I'm missing something.

Reply:
`null != id` is true and that's what we want to avoid. We should break out when it's null to avoid calling `null.parent`.

Reply:
`null != id` is true and that's what we want to avoid, as it continues the loop. We should break out when it's null to avoid calling `null.parent`.